Projects/Feature/BottleStorage/Interface/Sources/PingPongDetail/View/PhotoSharePingPongView.swift

```diff
@@ -123,7 +150,11 @@ private extension PhotoSharePingPongView {
   
   @ViewBuilder
   var questionText: some View {
-    if photoShareState == .notSelected || photoShareState == .waitingForPeer {
+    if photoShareState == .requireSelect(otherSelected: false)
+        || photoShareState == .requireSelect(otherSelected: true)
+        || photoShareState == .waitingOtherAnswer
+        || photoShareState == .myReject
+        || photoShareState == .otherReject {
```

Comment:
```suggestion
    if photoShareState != .disable && photoShareState != bothAgree {
```

반대 케이스가 좀 더 가독성 좋아보이는데 어떻게 생각해!?

Reply:
이거 반영해서 수정했어!